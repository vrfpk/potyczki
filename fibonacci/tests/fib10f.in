000000000262143000
