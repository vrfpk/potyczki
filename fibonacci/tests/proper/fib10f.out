2183354969897807250
